src/libs/SelectionScraper/index.js

```diff
@@ -105,6 +105,10 @@ const replaceNodes = (dom) => {
         if (htmlEngineUtils.isCommentTag(dom.attribs[tagAttribute])) {
             dom.children.push(new Element('br', {}));
         }
+    } else if (dom.name === 'div' && dom.children.length === 1 && dom.children[0].type !== 'text') {
+        // We are excluding divs that have only one child and no text nodes and don't have a tagAttribute.
+        // This is due to the fact that these divs are meaningless to us and are interfering with the HTML to Markdown conversion process.
```

Comment:
> and don't have a tagAttribute

Is that true? I can't see which condition this is referring to

Reply:
else condition

Reply:
Slight re-wording: 

> We are excluding divs that have only one child and no text nodes and don't have a tagAttribute to prevent additional newlines from being added in the HTML to Markdown conversion process.

Reply:
@Julesssss but yes we just explain what we have now so i update the comment